lib/kubernetes-deploy/rescue_krane_exceptions.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+require 'kubernetes-deploy/errors'
+
+module RescueKraneExceptions
```

Comment:
Sorry for only noticing now, but this module should be namespaced (under `KubernetesDeploy` because that's where it is in the directory structure)